### PR TITLE
Feature/organisation api

### PR DESCRIPTION
**Pull request by masahmad**

### CODE REVIEW

File: src/organisation/services/organisation.service.ts [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
 
+export const ENVIRONMENT = {
 
-const dummy = {
-  name: 'xyz solicitors Ltd',
-  address1: '10 Oxford Street',
-  town: 'London',
-  postcode: 'W1 ABC',
-
+  orgUri: '/api/organisation'
```
Comment (frontr-uk) [inline src/organisation/services/organisation.service.ts]: should this not all be part of the config file ?

File: src/organisation/store/actions/organisation.actions.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class LoadOrganisation {
 
 export class LoadOrganisationSuccess implements Action {
   readonly type = LOAD_ORGANISATION_SUCCESS;
-  constructor(public payload: Organisation) { }
+  constructor(public payload: any) { }
```
Comment (frontr-uk) [inline src/organisation/store/actions/organisation.actions.ts]: why did you set to any?

Comment (masahmad) [reply src/organisation/store/actions/organisation.actions.ts]: good spot - we're unsure on the model for organisation  - Example the format of the address property, is it a single string?  or multiple property as required for street, town,postcode - hence the api needs testing by posting a addresss - however there is a bug thats preventing this and this has been assigned to Tabby for now - will revisit and update <any>  in future

File: src/organisation/services/organisation.service.ts (hunk 1 above)
Comment (masahmad) [reply src/organisation/services/organisation.service.ts]: yes will make this change :)

File: src/organisation/containers/organisation/organisation.component.html [hunk 3]
```diff
@@ -16,7 +17,13 @@ <h1 class="govuk-heading-xl"> Organisation</h1>
               <td class="govuk-table__cell"></td>
            </tr>
         </tbody>
-     </table>
+     </table> -->
+
+     <h1>orgData from api:</h1>
+<pre>
+
```
[the remark this answers is not in the document]
Comment (masahmad) [reply src/organisation/containers/organisation/organisation.component.html]: yes I've commented out the grid and lef tthe json for now -  as the API is not returning the address in a format we need - its being checked by Tabby

**Pull request merged**